README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse-Project
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse-Project
+In this project, you will set up a "Hello world" repository.
+
+
+## Built With
+
+- HTML & CSS
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+- Create a repository
+- Create a new branch 
+- Add a .gitignore file.
+- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+- Add descriptive README file to your project.
+- Add 3 files (each one in a separate commit):
+     A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+     A .css file that includes one class with styles for your header (make it in your favorite color).
+     A test.md file with the text "This file should be ignored by git".
+- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.     
+
+
+## Authors
+
+👤 Ahmed Nazir Musah
+
+- GitHub: [@githubhandle](https://github.com/ahmednazirmusah)
+- Twitter: [@twitterhandle](https://twitter.com/ahmednazirmusah)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ahmed-nazir-musah-529956214?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3BaN%2F7ouJiQmaaZdym2t1Uxg%3D%3D)
```

Comment:
[OPTIONAL] Please try to update the README file, README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo.

Reply:
I have updated the Readme file as well. Thank you